sda-commons-starter/build.gradle

```diff
@@ -0,0 +1,22 @@
+dependencies {
```

Comment:
Very minor: We could sort this.

Reply:
What criteria do you like to sort the dependencies by? By name?